applications/asset_tracker_v2/src/ext_sensors/ext_sensors.c

```diff
@@ -157,26 +206,44 @@ int ext_sensors_init(ext_sensor_handler_t handler)
 	}
 #endif /* if defined(CONFIG_EXTERNAL_SENSORS_BME680_BSEC) */
 
-	if (!device_is_ready(accel_sensor.dev)) {
-		LOG_ERR("Accelerometer device is not ready");
+	if (!device_is_ready(accel_sensor_lp.dev)) {
+		LOG_ERR("Low-power accelerometer device is not ready");
 		evt.type = EXT_SENSOR_EVT_ACCELEROMETER_ERROR;
 		evt_handler(&evt);
-	} else {
-#if defined(CONFIG_EXTERNAL_SENSORS_ACTIVITY_DETECTION_AUTO)
+	} else if (IS_ENABLED(CONFIG_EXTERNAL_SENSORS_ACTIVITY_DETECTION_AUTO)) {
 		struct sensor_trigger trig = {
 			.chan = SENSOR_CHAN_ACCEL_XYZ,
 			.type = SENSOR_TRIG_THRESHOLD
 		};
 
-		int err = sensor_trigger_set(accel_sensor.dev,
+		int err = sensor_trigger_set(accel_sensor_lp.dev,
 					     &trig, accelerometer_trigger_handler);
 
 		if (err) {
 			LOG_ERR("Could not set trigger for device %s, error: %d",
-				accel_sensor.dev->name, err);
+				accel_sensor_lp.dev->name, err);
+			return err;
+		}
+	}
+
+	if (!device_is_ready(accel_sensor_hg.dev)) {
+		LOG_ERR("High-G accelerometer device is not ready");
+		/* Just use general accelerometer for both devices. */
```

Comment:
```suggestion
```

Reply:
You could update the documentation for `EXT_SENSOR_EVT_ACCELEROMETER_ERROR` instead of having the comment here. In `ext_sensors.h`.

`General event propagated when an error has occurred with any of the supported accelerometers.`

Something like that.

Reply:
done